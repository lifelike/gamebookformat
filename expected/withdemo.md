# Gamebook #


Turn to 1 to begin.
<a name="section1">

## 1 ##
 This gamebook contains a few sections tagged as demo, that will be included when building the example (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. Go on to [5](#section5)
 or [4](#section4)
 or [2](#section2)
. 
<a name="section2">

## 2 ##
 This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Go on to [4](#section4)
. 
<a name="section3">

## 3 ##
 This is the end. It is also included in the demo. 
<a name="section4">

## 4 ##
 This is the second demo section. You can go on to the end at [3](#section3)
 or alternative (not in demo) end at [2](#section2)
. 
<a name="section5">

## 5 ##
 This is the first demo section (not counting the start section). You can go on to the end at [3](#section3)
. 

