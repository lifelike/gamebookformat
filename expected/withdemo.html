<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'withdemo';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 This gamebook contains a few sections tagged as demo, that will be included when building the example (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. Go on to <a class="sectionref enabledlink" data-ref="5"
href="#section5">5</a> or <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a> or <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section2"
     >
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Go on to <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>
<div class="section" id="section3"
     >
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 This is the end. It is also included in the demo. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script>
<div class="section" id="section4"
     >
<div class="sectionnumber" id="para4">4</div>
<div class="sectiontext">
 This is the second demo section. You can go on to the end at <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> or alternative (not in demo) end at <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(4, document.getElementById('section4'));
}
</script>
<div class="section" id="section5"
     >
<div class="sectionnumber" id="para5">5</div>
<div class="sectiontext">
 This is the first demo section (not counting the start section). You can go on to the end at <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(5, document.getElementById('section5'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
